supersetapiclient/exceptions.py

```diff
@@ -1,5 +1,15 @@
 """Custom Exception."""
+from requests import HTTPError
 
 
 class NotFound(Exception):
     pass
+
+
+class ServerError(HTTPError):
```

Comment:
I agree with the unclear naming. The error is not necessarily caused by a bad request, but since that's how we use it now, I think `BadRequestError` is a good name.

Reply:
Do you want me to find a different name for `ComplexServerError` as well?

Reply:
Settling in `ComplexBadRequestError` in lack of better suggestion.